vignettes/getting-started.Rmd

```diff
@@ -19,24 +19,24 @@ knitr::opts_chunk$set(
 gimap performs analysis of dual-targeting CRISPR screening data, with the goal of aiding the identification of genetic interactions (e.g. cooperativity, synthetic lethality) in models of disease and other biological contexts. gimap analyzes functional genomic data generated by the pgPEN (paired guide RNAs for genetic interaction mapping) approach, quantifying growth effects of single and paired gene knockouts upon application of a CRISPR library. A multitude of CRISPR screen types can be used for this analysis, with helpful descriptions found in this review (https://www.nature.com/articles/s43586-021-00093-4). Use of pgPEN and GI-mapping in a paired gRNA format can be found here (https://pubmed.ncbi.nlm.nih.gov/34469736/). 
 
 ```{r}
-library(magrittr)
+library(dplyr)
```

Comment:
If we are going to use dplyr, we might as well load dplyr, which loads magrittr.

Reply:
Sounds good to me! Can you add this in too?